Makefile

```diff
@@ -0,0 +1,23 @@
+TRACK = 3
```

Comment:
**question:** Why makefile over tox? 

I don't mind either way as I like Make, just curious why the addition of a new tool?

Reply:
What's the feature diff?
They're not the same thing, sure. But we use them for the same purpose, namely aliasing bash scripts.

This conversation is non-blocking, I just don't see the advantage of using a new tool.